recipes/msamtools/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = '1.0.2' %}
+{% set build = '0' %}
+
+package:
+  name: msamtools
+  version: {{ version }}
+
+source:
+  url: https://github.com/arumugamlab/msamtools/releases/download/{{ version }}/msamtools-{{ version }}.tar.gz
+  sha256: 7d42a258aaabcbb238519cfa37874dd51602290cf418e7312933afd0eaa0a49e
+
+build:
+  number: {{ build }}
```

Comment:
I suggest to remove the Jinja2 variable and just write "0" directly. In my opinion, introducing a Jinja2 variable that is only used once just makes the recipe longer.

Reply:
Thanks for the suggestion. Will change this.